.buildkite/setup_and_upload_artifact.sh

```diff
@@ -35,7 +35,7 @@ mkdir -p dist
 buildkite-agent artifact download 'dist/*.pex' dist/
 buildkite-agent artifact download 'dist/*.whl' dist/
 buildkite-agent artifact download 'dist/*.tar.gz' dist/
-buildkite-agent artifact download 'dist/*.deb' dist/
+# buildkite-agent artifact download 'dist/*.deb' dist/
```

Comment:
I guess you were trying to comment out .dmg?

Reply:
🤦‍♂

Reply:
Good catch